frontend/js/app/nginx/access/form.ejs

```diff
@@ -46,6 +46,12 @@
 
                 <!-- Authorization -->
                 <div class="tab-pane" id="auth">
+                    <p>
+                        Basic Authorization via
+                        <a href="https://nginx.org/en/docs/http/ngx_http_auth_basic_module.html">
```

Comment:
Can you please add `target="_blank"` to prevent losing any data currently entered

Reply:
Good point. Should be added now!